### DISCUSSION TITLE
posts/032-weekly-report-dec-3/

### DISCUSSION THREAD

**Discussion by giscus** [Yamabuki-bakery/tasty-discussion#19] [Announcements]
# posts/032-weekly-report-dec-3/

🏆本週最難項目：畫一張畫的過程能出一本書

https://example.com

**Comment by PermeationLoop** [comment]
忘了說了，*CGイラストテクニック* 這個系列出版了很多期，大多都可以在 Z-lib 上找到．不過其他期的編寫質量有沒有上文中的 vol.10 那麼高，我就不知道了😶
